src/main/java/com/google/devtools/build/lib/includescanning/SpawnIncludeScanner.java

```diff
@@ -386,7 +386,7 @@ private static InputStream spawnGrep(
             outputs,
             LOCAL_RESOURCES);
 
-    actionExecutionContext.maybeReportSubcommand(spawn);
+    actionExecutionContext.maybeReportSubcommand(spawn, /* spawnRunner= */ "internal");
```

Comment:
Why is this sending `"internal"` but `ActionGraphFormatterCallback` is passing `null`?

Reply:
`SpawnIncludeScanner` actually runs a spawn, just not in any known runner (maybe `local` is more fitting than `internal` here, not sure), so I felt that a synthesized runner name made more sense.

`ActionGraphFormatterCallback` prints analysis-time information about an action and emitting spawn runners in that situation wouldn't make any sense.

Reply:
Can you add a comment to that effect?